docs/system-admin-guide/authentication/openid-providers/README.md

```diff
@@ -24,7 +24,88 @@ You can configure the following options.
 4. Enter the **Secret**.
 5. Press the blue **create** button.
 
-![Sys-admin-authentication-add-openid-provider](Sys-admin-authentication-add-openid-provider.png)
+
+
+
+
+## Google Workspace
+
+
+
+### Step 1: Create the OAuth consent screen
+
+1. Navigate to your GCP console.  (https://console.cloud.google.com/)
+2. Go to **APIs & Services** > OAuth consent screen.
+
+![image-20220215095201951](/home/ant0n/.config/Typora/typora-user-images/image-20220215095201951.png)
+
+
+
+3. Create a new project and a new app or edit an existing project and an existing app, setting the following fields (shall be Internal):
+   1. **App name** (e.g. EXAMPLE.COM SSO)
+   2. **User support email** (e.g. user-support@example.com)
+   3. **App domains** (at minimum, you must provide the Application home page - e.g. https://example.openproject.com)
+   4. **Authorized domains** (e.g. openproject.com)
+   5. **Developer Contact information** (e.g.  developer@example.com)
+   6. Click **SAVE AND CONTINUE** to proceed.
+
+![image-20220215095953811](/home/ant0n/.config/Typora/typora-user-images/image-20220215095953811.png)
+
+4. **Scopes** - Press **SAVE AND CONTINUE**
+5. **Summary** - Press **SAVE AND CONTINUE**
+
+
+
+### Step 2: Create the OAuth Client
+
+1. Under **APIs & Services**, go to **Credentials**.
+
+![image-20220215102049207](/home/ant0n/.config/Typora/typora-user-images/image-20220215102049207.png)
+
+2. Click **Create Credentials** and select **OAuth Client ID**.
+
+   1. When prompted for your **Application type**, choose **Web Application**.
+
+   2. Provide a **Name** for your application. (e.g. example-openproject-com)
+
+   3. Under Authorized redirect URIs, click **Add URI**, and provide your URI (e.g. [example.openproject.com]/auth/google/callback).
+
+   4. Click **CREATE** or **SAVE** .
+
+![image-20220215102234492](/home/ant0n/.config/Typora/typora-user-images/image-20220215102234492.png)
+
+After pressing **CREATE** you will get a pop-up window like the following
+
+- Note **Client ID**
+- Note **Client Secret**
+
+![image-20220215102658395](/home/ant0n/.config/Typora/typora-user-images/image-20220215102658395.png)
+
+
+
+### Step 3: Add Google as an OpenID Provider to OpenProject
+
+1. Login as Open Project Administrator
+2. navigate to -> *Administration* -> *Authentication* and choose -> *OpenID providers*.
+   1. **Name** Choose Google
+   2. **Display Name** (e.g. **EXAMPLE.COM SSO**)
+   3. **Identifier** (**Client ID** from step 2)
+   4. **Secret** (**Client Secret** from step 2)
+3. Press **Create**
+
+![image-20220215103036615](/home/ant0n/.config/Typora/typora-user-images/image-20220215103036615.png)
+
+4. The following green notification **Successful creation** should appear
+
+![image-20220215103344744](/home/ant0n/.config/Typora/typora-user-images/image-20220215103344744.png)
+
+
+
+
+
+
```

Comment:
Please avoid repeated newlines like that (keep them <= 2 newlines for spacing purposes)